### PR TITLE
Open Instruction Modal

### PR DESCRIPTION
**Pull request by jdemangeon**

### CODE REVIEW

File: src/utils/useLocalStorage.js [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+
+// From https://usehooks.com/useLocalStorage/
+
+export function useLocalStorage(key, initialValue) {
+    const [storedValue, setStoredValue] = useState(() => {
+        try {
+            const item = window.localStorage.getItem(key);
+            return item ? JSON.parse(item) : initialValue;
+        } catch (error) {
+            console.log(error);
```
Comment (Kmaschta) [inline src/utils/useLocalStorage.js:11]: Console.log spotted

File: src/App.js [hunk 2]
```diff
@@ -17,6 +16,10 @@ const Experiments = lazy(() =>
     import(/* webpackChunkName: "experiments" */ './screens/experiments/Experiments'),
 );
 
+const ProSimulation = lazy(() =>
+    import(/* webpackChunkName: "ProSimulation" */ './screens/simulation/Simulation'),
```
Comment (Kmaschta) [inline src/App.js:20]: :+1:

File: src/utils/useLocalStorage.js (hunk 1 above)
Comment (Kmaschta) [reply src/utils/useLocalStorage.js:11]: Ah, it is expected.. Use console.error then

File: src/utils/useLocalStorage.js [hunk 3]
```diff
@@ -0,0 +1,27 @@
...
+        }
+    });
+
+    const setValue = (value) => {
+        try {
+            const valueToStore = value instanceof Function ? value(storedValue) : value;
+            setStoredValue(valueToStore);
+            window.localStorage.setItem(key, JSON.stringify(valueToStore));
+        } catch (error) {
+            console.log(error);
```
Comment (Kmaschta) [inline src/utils/useLocalStorage.js:22]: console.error

File: src/screens/simulation/InstructionsButton.js [hunk 4]
```diff
@@ -207,6 +209,15 @@ const InstructionsButton = () => {
...
                     </CardContent>
                     <CardActions className={classes.actions}>
+                        {openOnMount && (
+                            <Button
+                                variant="outlined"
+                                color="primary"
+                                onClick={handleCloseAndDisableOpenOnMount}
+                            >
+                                Fermer &amp; Ne plus afficher
+                            </Button>
```
Comment (Kmaschta) [inline src/screens/simulation/InstructionsButton.js:219]:
```suggestion
                            <Button
                                variant="text"
                                color="primary"
                                onClick={handleCloseAndDisableOpenOnMount}
                            >
                                Ne plus afficher
                            </Button>
```

**Comment by jdemangeon** [comment]
Useful review applied

**Pull request merged**